pkg/service/browser.go

```diff
@@ -1016,6 +1016,7 @@ func waitForReady(browserCtx context.Context, cfg config.BrowserConfig) chromedp
 				(cfg.ReadinessNetworkIdleTimeout <= 0 || time.Since(start) < cfg.ReadinessNetworkIdleTimeout) &&
 				requests.Load() > 0 {
 				initialDOMPass = true
+				numSuccessfulCycles = 0
```

Comment:
I would only want this for queries, not all network requests

Reply:
okay, wasn't sure if there was overlap or not, will remove

Reply:
[a4ddb76](https://github.com/grafana/grafana-image-renderer/pull/884/commits/a4ddb761dee5433fb58c5d729cbab689181cc3b5)